src/script/event/EventRepository.ts

```diff
@@ -467,211 +455,4 @@ export class EventRepository {
     }
     return this.distributeEvent(event, source);
   }
-
```

Comment:
This entire piece of code was about saving events to the DB as they arrive. 
It has been broken down in the StorageMiddleware and split into specific event type handlers dealing with:
- edit events
- link preview events
- asset events